components/ircc/workflow/request-body.md

```diff
@@ -0,0 +1,7 @@
+The request body should be a JSON object containing the following fields:
+
+| Field           | Type          | Description   | Required |
+| --------------- | ------------- | ------------- | -------- |
+| `document`      | Object        | The schema object that represents the document being submitted. For example, the [IRCC schema](/ircc/schema). | Yes      |
+| `additionalInfo`| String        | Additional information related to the document. This field can hold a maximum of 1000 characters.    | No       |
```

Comment:
this field is available only when a use with NAU role is submitting a request to the PA to publish a record.